src/engine/client/cl_cgame.cpp

```diff
@@ -1287,6 +1287,21 @@ void CGameVM::QVMSyscall(int syscallNum, Util::Reader& reader, IPC::Channel& cha
 			});
 			break;
 
+		case CG_R_BATCHINPVS:
+			IPC::HandleMsg<Render::BatchInPVSMsg>(channel, std::move(reader), [this] (
+				const std::array<float, 3>& origin,
+				const std::vector<std::array<float, 3>>& posEntities,
+				std::vector<bool>& inPVS)
+			{
+				inPVS.resize(posEntities.size());
```

Comment:
Did you mean `reserve`?

Reply:
Right that's what I meant. Fixed.

What I'm not sure is, how do we are sure the size is always `0` here?

Reply:
The IPC system will construct an empty vector.

Reply:
OK! That's good to know.